### PR TITLE
rtmidi 2.1.1

### PR DESCRIPTION
**Pull request by MartinDelille**

### CODE REVIEW

File: Library/Formula/rtmidi.rb [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
   depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
 
   def install
-    system "autoconf"
-    system "./configure", "--prefix=#{prefix}"
-    system "make", "librtmidi.a"
-    lib.install Dir["*.a", "*.dylib"]
+    system "./autogen.sh"
```
Comment (DomT4) [inline Library/Formula/rtmidi.rb]: Was the `prefix` passing above not being used at all?

File: Library/Formula/rtmidi.rb [hunk 2]
```diff
@@ -1,23 +1,17 @@
 class Rtmidi < Formula
   desc "C++ classes that provide a common API for realtime MIDI input/output"
   homepage "https://www.music.mcgill.ca/~gary/rtmidi/"
-  url "https://github.com/thestk/rtmidi/archive/2.1.0.tar.gz"
-  sha256 "52e6822fc413d5d3963c5b7bfe412ed69233bb81a7f04d6097f5b56aafa28934"
-
-  bottle do
```
Comment (DomT4) [inline Library/Formula/rtmidi.rb]: Don't worry too much about it here but you don't need to wipe the `bottle do` block on updates.

Comment (MartinDelille) [inline Library/Formula/rtmidi.rb]: Ok thanks for the info!

File: Library/Formula/rtmidi.rb (hunk 1 above)
Comment (MartinDelille) [inline Library/Formula/rtmidi.rb]: Hmm good question, that was added by @robhardwick the original author of the formula. I will restore this behavior, but I wonder where can this variable come from? A `brew install rtmidi` parameter?

Comment (DomT4) [inline Library/Formula/rtmidi.rb]: If it's not being used, don't worry about it. There's no `make install` step to correlate to passing a prefix so not having to use one here wouldn't be _that_ unusual. Possibly just worth peeking into the `autogen` script to see if there's a `prefix` argument there.

**Comment by MartinDelille** [comment]
@DomT4 I just restore the prefix. I honnestly don't know what is this *prefix* made for.

**Comment by DomT4** [comment]
Alright, happy to accept as-is then unless we get some extra clarity on the `prefix` passing. It's not doing any harm, at least.

Merged in https://github.com/Homebrew/homebrew/commit/9e7015534a794e0e72f329cc8028b19a0be094d9. Thank you for another contribution to Homebrew @MartinDelille! :smiley_cat:

**Pull request closed without merging**